.rubocop.yml

```diff
@@ -6,6 +6,7 @@ AllCops:
   Exclude:
     - 'lib/truffle/date/format.rb' # crashes Rubocop in VariableForce
     - 'lib/truffle/date/delta/parser.rb' # generated code
+    - 'src/test/ruby/reenterValues.rb' # unusual test code
```

Comment:
Is there any significance to this filename format? If not, we should make it snake-case instead of camel-case to conform with our other Ruby files.

Reply:
Fixing.